composition/scripts/get-composition-version.mjs

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+import { fileURLToPath } from 'node:url';
+import path from 'node:path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compositionVersion = '{{$COMPOSITION__VERSION}}';
+
+// From pnpm v10+, hooks will not be supported.
```

Comment:
It is supported but we would need to whitelist modules that are allowed to run it.

Reply:
Updated the comment.